zerver/migrations/0262_mutedtopic_date_muted.py

```diff
@@ -14,6 +15,6 @@ class Migration(migrations.Migration):
         migrations.AddField(
             model_name='mutedtopic',
             name='date_muted',
-            field=models.DateTimeField(default=datetime.datetime(2020, 1, 1, 0, 0)),
+            field=models.DateTimeField(default=datetime.datetime(2020, 1, 1, 0, 0, tzinfo=utc)),
```

Comment:
Did you confirm this makes no change to the postgres schema, and thus this, rather than a migration change, is correct?  I don't remember how Django defaults work.

Reply:
Good question…I’m not even sure that checking this on the dev environment would reveal a problem here if there is one, since the dev environment has UTC as its local timezone.

Reply:
It looks like Django doesn’t use PostgreSQL defaults, and the schema is the same before and after:

```sql
CREATE TABLE zulip.zerver_mutedtopic (
    id integer NOT NULL,
    topic_name character varying(60) NOT NULL,
    recipient_id integer NOT NULL,
    stream_id integer NOT NULL,
    user_profile_id integer NOT NULL,
    date_muted timestamp with time zone NOT NULL
);
```